### PR TITLE
Fix doctests in ci

### PR DESCRIPTION
**Pull request by coolreader18**
- Fix doc comment for Interpreter
- Fix doctests in ci

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -73,6 +73,11 @@ opt-level = 3
 opt-level = 3
 lto = "thin"
 
+# ugly hack since doctests link against debug version of rustpython-vm even
+# when profile.test is optimized
```
Comment (youknowone) [inline Cargo.toml:77]: Though we usually use optimization and debug option exclusively, optimization is about optimization itself and debug is about debug information. So I think giving optimization option to test doesn't change it is a debug or not.

Comment (coolreader18) [reply Cargo.toml:77]: Right, and it does work fine unoptimized on our machines apparently, but I think there's some timeout in ci that kills the test process if it takes too long? And afaict it was linking against release in Rust 1.56, the update broke it a bit, this just brings it back to previous behavior

Comment (coolreader18) [reply Cargo.toml:77]: ach wait the tests aren't even fixed, dang it

Comment (youknowone) [reply Cargo.toml:77]: yep, I think it doesn't fix anything unless it is related to cache

Comment (coolreader18) [reply Cargo.toml:77]: ugh I don't know why I didn't consider that it was a cache issue

Comment (fanninpm) [reply Cargo.toml:77]: The cache action is reporting a cache *miss*, so I doubt that it's actually a cache issue.

Comment (fanninpm) [reply Cargo.toml:77]: Could https://github.com/rust-lang/rust/issues/92869 be related?

Comment (coolreader18) [reply Cargo.toml:77]: Ahh, probably

**Comment by coolreader18** [comment]
WOOOooooooo....! Passing!

**Comment by coolreader18** [comment]
Merging this so main turns green.

File: Cargo.toml [hunk 2]
```diff
@@ -71,7 +71,8 @@ opt-level = 3
 
 [profile.test]
 opt-level = 3
-lto = "thin"
+# https://github.com/rust-lang/rust/issues/92869
```
Comment (youknowone) [inline Cargo.toml:74]: this is totally unexpected effect

**Pull request merged**